index.js

```diff
@@ -162,7 +165,7 @@ async function checkCommit(commit = 'HEAD') {
   return _
     .chain(_.zip(files, results))
     .filter(pair => pair[1])
-    .map(pair => `${actualCommit.id}:${pair[0]}`)
+    .map(pair => `${actualCommit.id.slice(0, 7)}:${pair[0]}`)
```

Comment:
Instead of slicing, use `git rev-parse --short <object>` to be sure that in large projects it's not too short.